avocado/__init__.py

```diff
@@ -13,10 +13,19 @@
 # Author: Lucas Meneghel Rodrigues <lmr@redhat.com>
 
 
-__all__ = ['main', 'Test', 'VERSION', 'fail_on']
+__all__ = ['main',
+           'Test',
+           'VERSION',
+           'fail_on',
+           'skip',
```

Comment:
Yeah, out of the many options, I agree that skip* should live under the global avocado namespace.  The reason is that we call that the "test API", and skip* is really that.

Also, we're making unittest users at home.